README.md

```diff
@@ -70,16 +69,16 @@ These seem to be DOS aliases used with a LOAD command. (See SDK.BAT for an examp
 |`UTIL/X.BAT`|Older version of `NETON.BAT`?
 |`UTIL/X32FIX.EXE`|`FlashTek X-32 Fixup Utility` - Watcom util
 
-##Notable Tools
-###Deluxe Paint II Enhanced
+## Notable Tools
+### Deluxe Paint II Enhanced
 Barebones copies of Electronic Arts' famous pixel art application were present. Two versions, actually. Versions 2.1 and 2.3.
 
 <img src="http://i.imgur.com/G2YlHLu.jpg" width="49%" style="float:left"/>
 <img src="http://i.imgur.com/zqZWWbK.jpg" width="49%" style="float:right"/>
 <div style="clear:both"></div>
 
-###DK.EXE
-This, from what I recall, is an in-house LucasArts image 'Swiss Army Knife' tool. Among it's powers: it could reduce the number of colors in an image, using dithering, or adapt an image to fit an existing palette.
+### DK.EXE
+This, from what I recall, is an in-house LucasArts image 'Swiss Army Knife' tool. Among its powers: it could reduce the number of colors in an image, using dithering, or adapt an image to fit an existing palette.
```

Comment:
A typo I certainly would have picked up on nowadays. Had that one hammered into my head a year after I wrote it. ;) (That and not using "80's", just 80s, etc.)